src/ExadelMentorship.WebApi/ExadelMentorship.WebApi.csproj.user

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This type of files should be `gitignore`d

Reply:
added csproj.user to gitignore